shell/platform/darwin/ios/framework/Source/FlutterViewController_Internal.h

```diff
@@ -31,7 +31,7 @@ extern NSNotificationName const FlutterViewControllerShowHomeIndicator;
 
 @property(nonatomic, readonly) BOOL isPresentingViewController;
 @property(nonatomic, readonly) BOOL isVoiceOverRunning;
-@property(nonatomic) FlutterKeyboardManager* keyboardManager;
+@property(nonatomic, strong) FlutterKeyboardManager* keyboardManager;
```

Comment:
Isn't `strong` an ARC annotation?

Reply:
Yes, whoops.

Reply:
Interesting, switching all the `strong`s (outside the public headers) to `retain`s exposed a few more missing `dealloc`s.